catroid/src/main/java/org/catrobat/catroid/content/bricks/brickspinner/PickableMusicalInstrument.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2020 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.content.bricks.brickspinner;
+
+import org.catrobat.catroid.common.Nameable;
+import org.catrobat.catroid.utils.Utils;
+
+import java.io.Serializable;
+
+public class PickableMusicalInstrument implements Nameable, Serializable {
+
```

Comment:
you missed to add `serialVersionUID` attribute for this class, please just add this line
 `private static final long serialVersionUID = 1L;`

Reply:
done, thanks for the review :)